src/Raven.Server/Documents/AI/Embeddings/StreamedEmbeddingValue.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.IO;
+using Sparrow.Server;
+
+namespace Raven.Server.Documents.AI.Embeddings;
+
+public class StreamedEmbeddingValue : IEmbeddingValue
```

Comment:
Only reason it exists is that this is an attachment. But, we can deal with that more easily by materializing the attachment or passing a `Span<byte>` from the memory directly, instead of like that. 
We _know_ what the values are.